### DISCUSSION TITLE
Move the touch button?

### DISCUSSION THREAD

**Discussion by lobau** [Schneegans/Fly-Pie#186] [Ideas & Feature Discussions]
I love the idea of keeping the touch button on screen at all time, but right now, it's always in the center of the screen. Is there a way to move the touch button somewhere else when you have "Show Touch Button" enabled?

**Comment by Schneegans** [comment]
Long press it :)

Maybe we have to show a hint somewhere. This seems to be hard to discover...

**Reply by lobau** [reply]
Haha awesome thanks! Yeah maybe when you hover the toggle in the UI the help text below should explain it?
